QuestionAnswering/QA_SEQ2SEQ/deployment/fastapi_app/services/translate/data_processing/ontospecies/compact2verbose.py

```diff
@@ -136,7 +136,7 @@ def _try_convert_species_haspropertyabstract_triple(self, pattern: GraphPattern)
             assert isinstance(pattern, TriplePattern)
             assert pattern.subj == "?Species"
             assert len(pattern.tails) == 1
-            
+
             predicate, obj = pattern.tails[0]
             assert predicate == "?hasPropertyName"
             assert obj == "?PropertyName"
```

Comment:
See comment above. Several occurrences in this file.

Reply:
I've replaced the `assert` statement with `raise ValueError(...)`.